installation/production_deb.rst

```diff
@@ -60,6 +60,10 @@ Before starting the database let's change its access permissions. By default the
 
 **4. Start database servers, create Citus extension**
 
+.. note::
+
+  Before starting the cluster, be aware that by default Citus reports anonymous usage information to Citus Data company servers. To learn more about this, including how to opt out, see :ref:`phone_home`.
```

Comment:
usage statistics makes me feel that we're collecting information on how the cluster is being used like how many queries or utilization or things like that, where as we're just collecting information about the cluster i.e. how many nodes, how many tables etc. What do you think about changing the notice to:

`Please note that Citus reports anonymous information about your cluster to the Citus Data company servers. To learn more about what information is collected and how to opt out of it, see phone home.`